code-challenges.test.js

```diff
@@ -13,55 +13,131 @@
+const fibonacci = (number) => {
+  //The number passed must be greater than 2
+  if (number>2){
+    //create a variable named sequence that holds two numbers [1, 1]
+    //that begins our fibonacci sequence
+    sequence=[1, 1]
+    //create a loop with variable i starting at the number 1 and will
+    //increment positively until it reaches the number minus 1
+    for(i=1; i<number-1; i++){
+      //<sequence[i-1]> is the position of the previous number in the array "sequence"
+      //<sequence[i]> is the position of the current number in the array "sequence"
+      //We add these two numbers and use the .push() method to attach these numbers
+      //to the end of the "sequence" array until the loop ends
+      sequence.push(sequence[i-1] + sequence[i])
+    }
+  } else {
+    //If a value other than a number greater than 2 was passed as an argument the following will trigger
+    return 'Something went wrong'
+  }
+  //return the fibonacci sequence the length of the number passed
+   return sequence
+}
 
 // --------------------2) Create a function that takes in an object and returns an array of the values sorted from least to greatest.
 // Hint: Check out this resource: Object.values() https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values
 
 // a) Create a test with expect statements for each of the variables provided.
+describe("leastGreatest", () => {
+  it("takes in an object and returns an array of the values sorted from least to greatest", () => {
+    const studyMinutesWeek1 = {
+      sunday: 90,
+      monday: 30,
+      tuesday: 20,
+      wednesday: 15,
+      thursday: 30,
+      friday: 15,
+      saturday: 60
+    }
+    // Expected output: [15, 15, 20, 30, 30, 60, 90]
+    const studyMinutesWeek2 = {
+      sunday: 100,
+      monday: 10,
+      tuesday: 45,
+      wednesday: 60,
+      thursday: 20,
+      friday: 15,
+      saturday: 65
+    }
+    // Expected output: [10, 15, 20, 45, 60, 65, 100]
+
+    expect(leastGreatest(studyMinutesWeek1)).toEqual([15, 15, 20, 30, 30, 60, 90])
+    expect(leastGreatest(studyMinutesWeek2)).toEqual([10, 15, 20, 45, 60, 65, 100])
+  })
+})
 
-const studyMinutesWeek1 = {
-  sunday: 90,
-  monday: 30,
-  tuesday: 20,
-  wednesday: 15,
-  thursday: 30,
-  friday: 15,
-  saturday: 60
-}
-// Expected output: [15, 15, 20, 30, 30, 60, 90]
-
-const studyMinutesWeek2 = {
-  sunday: 100,
-  monday: 10,
-  tuesday: 45,
-  wednesday: 60,
-  thursday: 20,
-  friday: 15,
-  saturday: 65
-}
-// Expected output: [10, 15, 20, 45, 60, 65, 100]
 
 // b) Create the function that makes the test pass.
+//create a function named leastGreatest that takes an object as a parameter
+const leastGreatest = (object) => {
+  //create a variable named sortedNumbers that hold the value of the 
+  //objects value using Object.values()
+  //sort the numbers in ascending order using the .sort method
+  //within the .sort() method we need to distinguish that we want
```

Comment:
Awesome! Definitely want to take into consideration how we want to sort and you did it perfectly!